setup/context/commands.go

```diff
@@ -393,35 +430,60 @@ func Endpoint(c *Context, syntax string, insecure, dev, quiet bool) error {
 }
 
 func Plan(c *Context, args []string) error {
-	if err := checkDockercfg(c, false); err != nil {
+	//TODO: Declare io elsewhere and pass in for testing
+	io := DefaultFileIO{}
+	if err := checkDockercfg(c, false, io); err != nil {
 		return err
 	}
 
 	args = append([]string{"plan"}, args...)
 	return Terraform(c, args)
 }
 
-func checkDockercfg(c *Context, force bool) error {
+func checkDockercfg(c *Context, force bool, io FileIO) error {
 	path := fmt.Sprintf("%s/dockercfg", c.InstanceDir)
 
-	// if file doesn't exist
-	if _, err := os.Stat(path); err != nil {
+	// if file does not exist
+	if _, err := io.Stat(path); err != nil {
+
+		// write empty dockercfg file
+		if err := io.WriteFile(path, []byte("{}"), 0666); err != nil {
```

Comment:
[nit] the file permission here is `0666` --- I'd recommend changing it to `0660`, not sure why `everyone` would need to have read/write access to an auth file, esp on a multi-user system (this was like this prior to your PR, btw, missed it the first time!)